internal/xds/translator/translator.go

```diff
@@ -245,3 +269,11 @@ func findXdsRouteConfig(tCtx *types.ResourceVersionTable, name string) *routev3.
 
 	return nil
 }
+
+type addXdsClusterArgs struct {
+	name         string
+	destinations []*ir.RouteDestination
+	tSocket      *corev3.TransportSocket
+	isHTTP2      bool
+	isStatic     bool
```

Comment:
prefer to change these to 
```
ProtocolType  //enum of  HTTP2/TCP/HTTP...
ClusterType  // enum of Static/EDS
```

@arkodg WDYT?

Reply:
ok, I refactor these code.